### ISSUE TITLE
Error when re-build edgetpu library

### ISSUE THREAD

**Issue by AlexTraan** [google-coral/edgetpu#87]
Hi Team,

I updated the latest version, then re-build but faced below issue:
"WARNING: Download from https://github.com/bazelbuild/bazel-skylib/releases/download/0.9.0/bazel_skylib-0.9.0.tar.gz failed: class javax.net.ssl.SSLProtocolException Read timed out
ERROR: An error occurred during the fetch of repository 'bazel_skylib':
   java.io.IOException: Error downloading [https://github.com/bazelbuild/bazel-skylib/releases/download/0.9.0/bazel_skylib-0.9.0.tar.gz] to /home/alex/.cache/bazel/_bazel_alex/190b849a4adec7cd86ce673d6938b9ea/external/bazel_skylib/bazel_skylib-0.9.0.tar.gz: Read timed out
ERROR: Analysis of target '//src/cpp/pipeline:models_test' failed; build aborted: no such package '@bazel_skylib//lib': java.io.IOException: Error downloading [https://github.com/bazelbuild/bazel-skylib/releases/download/0.9.0/bazel_skylib-0.9.0.tar.gz] to /home/alex/.cache/bazel/_bazel_alex/190b849a4adec7cd86ce673d6938b9ea/external/bazel_skylib/bazel_skylib-0.9.0.tar.gz: Read timed out
INFO: Elapsed time: 28.448s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
    currently loading: @org_tensorflow//tensorflow
Makefile:106: recipe for target 'tests' failed
make: *** [tests] Error 1
"

If i copy and past this [https://github.com/bazelbuild/bazel-skylib/releases/download/0.9.0/bazel_skylib-0.9.0.tar.gz]  to web browser, i can download it, so i don't thing this is network firewall to block this download. 

Can you help?

**Comment by Namburger** [comment]
Sorry, what are you rebuilding (command please)? I think GitHub was down yesterday for a light period and then became a little slow, it's very possibly due to that

**Comment by AlexTraan** [comment]
"what are you rebuilding (command please)? "
>> i build whole edgetpu repos, the command is "~/edgetpu$ make" 

"I think GitHub was down yesterday for a light period and then became a little slow, it's very possibly due to that"
>> let me try again

**Comment by AlexTraan** [comment]
For now, it's Ok when i build whole edgetpu repos.
But if i build only posenet_decoder.so in "/edgetpu/src/cpp/posenet" with command "bazel build posenet_decoder.so", i faced other issue as below:
"ERROR: /home/alex/github/edgetpu/src/cpp/posenet/BUILD:91:1: C++ compilation of rule '//src/cpp/posenet:posenet_decoder.so' failed (Exit 1) gcc failed: error executing command /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer '-std=c++0x' -MD -MF ... (remaining 35 argument(s) skipped)"

**Comment by AlexTraan** [comment]
Now, this is OK
